symfony/webpack-encore/0.9/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+    // directory where all compiled assets will be stored
+    .setOutputPath('web/build/')
+    // what's the public path to this directory (relative to your project's document root dir)
+    .setPublicPath('/build')
+    // empty the outputPath dir before each build
+    .cleanupOutputBeforeBuild()
+    // allow sass/scss files to be processed
+    .enableSassLoader()
```

Comment:
> To push in this direction, Bootstrap changed its default implementation from SASS to LESS in version 3.
Bootstrap has always used Less. They had a Sass-alternative package for the 3rd version. However as you say, they fully switched to the Sass framework starting at version 4. Anyway, Sass has won the game. Less will be around for legacy projects but any new project now should rely on Sass.